web/src/drag/drag-slice.test.ts

```diff
@@ -15,7 +15,11 @@ import {
   LogicalLocation,
 } from "./DragStateTypes";
 import { FakeDroppableMonitor } from "./__test_util__/FakeDroppableMonitor";
-import { configureStore, getDefaultMiddleware } from "@reduxjs/toolkit";
+import {
+  configureStore,
+  EnhancedStore,
```

Comment:
These don't look like your names, but the mix of camel case and Pascal case seems strange. Do you know if it's intentional and/or has any significance?

Reply:
PascalCase is a class or interface, camelCase is a method